### DISCUSSION TITLE
Any further crossref system prefixes planned?

### DISCUSSION THREAD

**Discussion by jimjam-slam** [quarto-dev/quarto-cli#10025] [Q&A] [answered]
### Description

The [docs page](https://quarto.org/docs/authoring/cross-references.html) for the global crossref system has a list of reserved prefixes:

> The reserved prefixes are: `fig`, `tbl`, `lst`, `tip`, `nte`, `wrn`, `imp`, `cau`, `thm`, `lem`, `cor`, `prp`, `cnj`, `def`, `exm`, `exr`, `sol`, `rem`, `eq`, `sec`.

Are there plans to substantially expand these? @andrewpbray and I are talking about moving our upcoming extension from using a prefixed attribute to using a prefixed ID, `#cr-`. We want to make sure we're not going to make headaches for our users by booking a prefix you were planning on using!

**Comment by mcanouil** [comment, marked as answer]
I would say yes:

- https://github.com/quarto-dev/quarto-cli/issues/3979

FWIW, with the custom cross-reference system, the list is quite infinite.

Note that Quarto tends to use three letters prefix, so if you choose something else, you are less likely to overlap with Quarto.
